docs/sources/architecture/internationalization.md

```diff
@@ -109,6 +109,10 @@ thing-type.yahooweather.weather.description = Stellt verschiedene Wetterdaten vo
 thing-type.config.yahooweather.weather.location.label = Ort
 thing-type.config.yahooweather.weather.location.description = Ort der Wetterinformation.
 
+thing-type.configGroup.yahooweather.weather.group1.label = a label
+thing-type.configGroup.yahooweather.weather.group1.description = a description
```

Comment:
Regarding to my previous commit I would change it to `thing-type.config.yahooweather.weather.group1.label = a label` and `thing-type.config.yahooweather.weather.group1.description = a description`